tests/system/load/test_large_tables.py

```diff
@@ -76,19 +76,21 @@ def test_index_repr_large_table():
 
 def test_to_pandas_batches_large_table():
     df = bpd.read_gbq("load_testing.scalars_10gb")
+    # double size to catch 10gb limit errors
+    for column in df:
+        df[column + "_2"] = df[column]
     # df will be downloaded locally
     expected_row_count, expected_column_count = df.shape
 
     row_count = 0
-    # TODO(b/340890167): fix type error
-    for df in df.to_pandas_batches():  # type: ignore
-        batch_row_count, batch_column_count = df.shape
+    for pdf in df.to_pandas_batches(page_size=500000):
```

Comment:
I think `page_size` can counter-intuitively result in much lower throughput, as it requires the use of the BigQuery REST API, not the BigQuery Storage API (which has some tunables but not exactly based on number of rows per page). I'd prefer we wait to add this parameter until we get an explicit request for it.

Reply:
I will change it to only download 3 batches, so that we don't need to use large pages just to get the test to finish in a reasonable time. Then we can delay putting in the page_size parameter as you suggest.